src/components/PDFView/index.js

```diff
@@ -283,7 +284,7 @@ class PDFView extends Component {
                     }) => this.setState({containerWidth: width, containerHeight: height})}
                 >
                     <Document
-                        error={<Text style={[styles.textLabel, styles.textLarge]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
+                        error={<Text style={[this.props.errorLabelStyles, cursor.cursorAuto]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
```

Comment:
I had to add cusrorAuto, because I'm not sure if you have noticed, but the text is still clickable to download the PDF, since the PDF is not corrupt, it's just the preview that is corrupt, I was even thinking that we might another way to do this.